VimeoNetworking/Sources/Models/Subscription.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  self.swift
+//  Pods
+//
+//  Created by Lim, Jennifer on 2/8/17.
+//
+//
+
+import Foundation
+
+public class Subscription: VIMModelObject
+{
+    // MARK: - Properties
+    
+    public var comment: NSNumber?
+    public var credit: NSNumber?
+    public var like: NSNumber?
+    public var mention: NSNumber?
+    public var reply: NSNumber?
+    public var follow: NSNumber?
+    public var videoAvailable: NSNumber?
+    public var vodPreorderAvailable: NSNumber?
+    public var vodRentalExpirationWarning: NSNumber?
+    public var accountExpirationWarning: NSNumber?
+    public var share: NSNumber?
+    
+    public var toDictionary: [NSObject: AnyObject]
```

Comment:
We need to keep it as a `var` because this dictionary can mutate if we update one of those var above.
I can't change it to be more specific about the type because, i need to `toDictionary` property in our request that takes a `anyobject`. Otherwise, i will have to `flatMap` on that dictionary :(